.travis.yml

```diff
@@ -21,58 +18,40 @@ script:
       else
         ./configure $ARG && make -j2 V=0;
       fi
-    - if [ "$VALGRIND" = "true" ]; then
-        make run-valgrind V=0;
-      elif [ "$BUILD_TYPE" != "ios" ]; then
+    - if [ "$BUILD_TYPE" != "ios" ]; then
         make -j2 check-am V=0;
       fi
 
 # Silence gcov standard output since it produces way too much output
 after_success:
     - if [ "$COVERAGE" = "true" ]; then
-        coveralls --gcov /usr/bin/gcov-4.6 --exclude src/ext
-                  --exclude example --exclude third_party > /dev/null;
+        coveralls --gcov /usr/bin/gcov-5 --exclude src/ext -b .
+                  --exclude example --exclude third_party
+                  --exclude include/measurement_kit/ext > /dev/null;
       fi
 
 after_failure:
     - if [ -f "test-suite.log" ]; then
         cat test-suite.log;
       fi
 
-# Only measure coverage with clang for now because I cannot manage to
-# convince coveralls to find all sources when using gcc-4.8 and gcov-4.8
-# (For more context see measurement-kit/measurement-kit#255)
 matrix:
   include:
 
     - sudo: false
-      compiler: clang-3.6
-      env: OS="12.04" VALGRIND=true VER=3.6
-      addons:
-        apt:
-          packages:
-            - clang-3.6
-            - llvm-3.6-dev
-            - valgrind
-          sources:
-            - llvm-toolchain-precise-3.6
-            - ubuntu-toolchain-r-test
```

Comment:
This entry was removed; the same functionality is now implemented using gitlab-ci.